components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -145,8 +145,7 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
 
     combineLatest([this.isIAMv2$, team$]).pipe(
       takeUntil(this.isDestroyed),
-      filter(([_isV2, team]) => team !== undefined),
-      distinctUntilChanged(([isV2, team]) => isV2 === isV2 && team === team)
```

Comment:
The fix also works if I change `distinctUntilChanged` to `distinctUntilChanged()`

I'm not 100% sure what the function offers over the filter 🤔

Reply:
The `distinctUntilChanged` function tests the previously seen value with the current one. The event only moves on to the subscribe block if the current data is different than the previous data. 

I am not sure why removing the line fixed the problem. I am going to test it locally.

Reply:
I found the problem. I was using the `distinctUntilChanged` function wrong. With the `distinctUntilChanged` you have two parameters (`distinctUntilChanged((current, previous) => test if the current the same as previous)`). I only had it with one parameter. The below code is working for me. 
```
distinctUntilChanged(([isV2A, teamA], [isV2B, teamB]) =>
      isV2A === isV2B && teamA === teamB)
```

Reply:
Does this mean we need to revisit other places where we are using `distinctUntilChanged`, @lancewf ?

Reply:
I don't think we have to because this is the only place where we use arguments. The only reason we are using arguments here is that the value is an array. If the value is just a string then you don't need any arguments.